### ISSUE TITLE
Gradle Check style flagging constants naming as error

### ISSUE THREAD

**Issue by jerroldlam** [nus-cs2113-AY2021S1/forum#77]
As per coding standards, constants should be in all caps separated by an underscore "_". However, for my team's tp, the checkstyle has flagged all our constants as error with the following 
```
Abbreviation in name 'STARTING_SEMESTER_INDEX' must contain no more than '2' consecutive capital letters.
```

Inside the code it looks like this 
```
private final int STARTING_SEMESTER_INDEX = 1;
```

I have tried running the script again when I have deleted the private and it is still appearing as an error. Unsure of how to proceed, whether to change the names for constants to fit the check style or ignore the check style warnings.

**Comment by Zhi-You** [comment]
This is something new to me too as I thought the code that you have written for your constant is fine. I tried what you did and the check gave me the same warning. However, when I added 'static' before 'final' the check passed. So, I believe that constants should have the 'static' keyword in it and hence the check only passes such cases. 
Also, if you try to refactor something in your code into a constant, the IDE will automatically declare it with the 'static' key word in it, like so:
![image](https://user-images.githubusercontent.com/49060098/95012506-b56cb600-066b-11eb-9cf0-fc38bbf5f438.png)

Below are some useful information I found about what static does and I hope this might explain why we need 'static' when declaring our constants. 
[StackOverflow (see the comment with 104 upvotes)](https://stackoverflow.com/questions/8093230/why-are-java-constants-declared-static#:~:text=If%20a%20constant%20is%20not,of%20the%20constant%20per%20object)
